File: Installation/setpath.m
%***************************************************************************                               
%                     Molecular Dynamics Potentials (MDP)
%                            CESMIX-MIT Project  
%  
% Contributing authors: Ngoc-Cuong Nguyen (cuongng@mit.edu, exapde@gmail.com)
%***************************************************************************

% Add MDP to Matlab search path
addpath(char(MDPpath + "src/Matlab/Gencode"));
addpath(char(MDPpath + "src/Matlab/Preprocessing"));
addpath(char(MDPpath + "src/Matlab/Postprocessing"));
addpath(char(MDPpath + "src/Matlab/Fitting"));

% Set Matlab's PATH enviroment variable so that Exasim can call external packages    
setenv('PATH', '/usr/local/bin:/usr/bin:/opt/local/bin:/bin:/usr/sbin:/sbin');

File: Installation/setpath.jl
# Add MDP to Julia search path
push!(LOAD_PATH, MDPpath * "src/Julia");
push!(LOAD_PATH, MDPpath * "src/Julia/Gencode");
push!(LOAD_PATH, MDPpath * "src/Julia/Preprocessing");
push!(LOAD_PATH, MDPpath * "src/Julia/Postprocessing");
push!(LOAD_PATH, MDPpath * "src/Julia/Potential");
push!(LOAD_PATH, MDPpath * "src/Julia/Optimization");

# Set Julia's PATH enviroment variable so that MDP can call external programs
ENV["PATH"] = "/usr/local/bin:/usr/bin:/opt/local/bin:/bin:/usr/sbin:/sbin";

using Preprocessing, Potential, Optimization, Integration 

# load Snap library 
Potential.loadsnap(MDPpath)



